spec/requests/submissions_download_spec.rb

```diff
@@ -8,7 +8,7 @@
 
     get download_task_submission_path(mock_task_id, mock_submission_id)
 
-    expect(response.status).to eql 307
-    expect(response).to redirect_to('http://s3.example.com/example.xls')
+    expect(response.body).to eq('')
```

Comment:
Is this representative of what really happens (i.e. is it a chunked HTTP download) or is it a test artefact that the body's blank?

Reply:
It's the test artefact. It's not a great test, but I couldn't think of a better way given the AWS library. Definitely open to suggestions.

Reply:
I'd either test against a string like "fake body" that you pre-allow somewhere else or just delete the line altogether. The `octet-stream` assertion is sufficient to indicate that a download is happening.